### DISCUSSION TITLE
Create issue comment with custom author username

### DISCUSSION THREAD

**Discussion by GnomedDev** [community/community#60442] [Enterprise]
### Select Topic Area

Question

### Body

I'm looking to develop a tool to completely clone a Github repository (including issues and PRs) to a different (blank) repo, to allow for cleaner "post-archive" forks to exist.

It seems like Github has support for "mannequin" accounts, as seen in https://github.com/llvm/llvm-project/issues/1138#issuecomment-980855703. It seems like these are related to Github Enterprise Importer, but I would simply like a way to create these mannequin issue comments (and maybe even issues) via the API.

**Comment by jbakerdev** [comment]
@GnomedDev did you ever figure out a solution for this?

**Reply by GnomedDev** [reply]
Nope.
